src/test/java/spoon/test/spoonifier/SpoonifierTest.java

```diff
@@ -91,6 +91,7 @@ public void testSpoonifier() throws ClassNotFoundException, MalformedURLExceptio
 		testSpoonifierWith("src/test/java/spoon/test/spoonifier/testclasses/ArrayRealVector.java", i++);
 		testSpoonifierWith("src/test/java/spoon/test/prettyprinter/testclasses/FooCasper.java", i++);
 		testSpoonifierWith("src/test/java/spoon/test/prettyprinter/testclasses/Rule.java", i++);
+		testSpoonifierWith("src/test/resources/UnnamedPackage.java", i++);
```

Comment:
I added this test just to ensure that classes with `unnamed packages` can also be spoonified.